axelor-account/src/main/resources/views/Invoice.xml

```diff
@@ -741,6 +741,10 @@ InvoiceSet.length > 0))"
           <field name="legalNotice" requiredIf="financialDiscount" widget="html" colSpan="12"
             showTitle="false"/>
         </panel>
+        <panel name="invoiceProductStatementPanel" title="Invoice Product Statement"
+          colSpan="12" if="__user__.activeCompany?.accountConfig?.displayItemsCategoriesOnPrinting">
```

Comment:
we should check the company from the invoice here, not from the user, so please use company from the invoice and use a `showIf` condition instead of `if`